docs/mp/health/01_introduction.adoc

```diff
@@ -114,24 +126,360 @@ microservice reports itself as ready.
+    readyTime.set(System.currentTimeMillis()); // <3>
+  }
+
+  private boolean isStarted() { // <4>
+    return Duration.ofMillis(System.currentTimeMillis() - readyTime.get()).getSeconds() >= 8;
+  }
+}
+----
+<1> Annotation indicating that this is a startup health check.
+<2> Build the `HealthCheckResponse` with status `UP` after eight seconds, else `DOWN`.
+<3> Record the time at startup of Helidon; the application will declare itself as started eight seconds later.
+<4> Become ready after 5 seconds.
+
+[source,bash]
+.Build and run the application.  Issue the curl command with -v within five seconds and you will see that the application has not yet started:
+----
+> curl -v  http://localhost:8080/health/started
+----
+
+[source,json]
+.HTTP response:
+----
+...
+< HTTP/1.1 503 Service Unavailable // <1>
+...
+{
+  "status": "DOWN",
+  "checks": [
+    {
+      "name": "StartedCheck",
+      "status": "DOWN",
+      "data": {
+        "time": 1566399775700
+      }
+    }
+  ]
+}
+----
+<1> The HTTP status is `503` since the application has not started.
+
+[source,bash]
+.After eight seconds you will see the application has started:
+----
+> curl -v http://localhost:8080/health/started
+----
+
+[source,json]
+.JSON response:
+----
+...
+< HTTP/1.1 200 OK // <1>
+...
+{
+  "status": "UP",
+  "checks": [
+    {
+      "name": "StartedCheck",
+      "status": "UP",
+      "data": {
+        "time": 1566399775700
+      }
+    }
+  ]
+}
+----
+<1> The HTTP status is `200` indicating that the application is started.
+
+When using the health check URLs, you can get the following health check data:
+
+* liveness only - http://localhost:8080/health/live
+* readiness only -  http://localhost:8080/health/ready
+* startup checks only - http://localhost:8080/health/started
+* all health check data -  http://localhost:8080/health
+
+[source,bash]
+.Get all the health check data, including custom data:
+----
+> curl http://localhost:8080/health
+----
+
+[source,json]
+.JSON response:
+----
+{
+  "status": "UP",
+  "checks": [
+    {
+      "name": "LivenessCheck",
+      "status": "UP",
+      "data": {
+        "time": 1566403431536
+      }
+    },
+    ...
+}
+----
 
-You can see all the defaults by accessing any Helidon MP microservice's `/health/live` endpoint
-and viewing the response.
+Full example code is available link:https://github.com/oracle/helidon/tree/master/examples[here].
 
-==== Custom Health Checks
-Add your own liveness, readiness, or startup checks by adding a Java class for each check.
-Each custom check must implement the `HealthCheck` interface, and you add either the `@Liveness`,
- `@Readiness`, or `@Startup` annotation to the class.
 
-=== Next Steps
-Add custom health checks to your own microservices.
+== Additional information
 
-The
-<<mp/guides/04_health.adoc, Helidon MP Health Check Guide>> shows how to create a
-sample project and add custom health checks.
\ No newline at end of file
+No additional information provided
```

Comment:
I would delete the whole "Additional Information" section if its empty.

Reply:
Added information.

Reply:
Although a link to microprofile.io could be useful, it's more useful to add links to MP Health 4.0 and MP Health API repository.

Reply:
There is already a link in the Usage description. Should it be duplicated here?

Reply:
Can you add a link to Health javadocs?

Reply:
Done)